src/odbc/connectparams.c

```diff
@@ -556,7 +560,7 @@ odbc_build_connect_string(TDS_ERRS *errs, TDS_PARSED_PARAM *params, char **out)
 
 #if !HAVE_SQLGETPRIVATEPROFILESTRING
 
-#ifdef _WIN32
+#if defined(_WIN32)  &&  !defined(TDS_NO_DM)
```

Comment:
Personally I would avoid this, Windows has a standard driver manager which should be used